tests/conftest.py

```diff
@@ -0,0 +1,7 @@
+import py
+import pytest
+
+
+@pytest.fixture
+def original_shared_datadir(request):
+    return py.path.local(request.fspath.dirname).join("data")

```

Comment:
I didn't know about `fspath` attribute of the `request` fixture! :+1: 

> the file system path of the test module which collected this test
https://docs.pytest.org/en/latest/reference.html#_pytest.fixtures.FixtureRequest.fspath